src/DynamoCoreWpf/Controls/InstalledPackagesControl.xaml

```diff
@@ -68,7 +68,7 @@
         <ScrollViewer Grid.Row="1"  
             VerticalScrollBarVisibility="Auto" 
             Background="{StaticResource PreferencesWindowBackgroundColor}" 
-            MaxHeight="290">
+            MaxHeight="246">
```

Comment:
Is there any explanation for choosing this value or is it chosen by trial and error?